x-pack/plugins/alerts/server/alerts_client/alerts_client.ts

```diff
@@ -682,6 +686,8 @@ export class AlertsClient {
     { id, data }: UpdateOptions,
     { attributes, version }: SavedObject<RawAlert>
   ): Promise<PartialAlert> {
+    this.alertTypeRegistry.ensureAlertTypeEnabled(attributes.alertTypeId);
```

Comment:
**nit:** since `updateAlert` is only called from `updateWithOCC` and `updateWithOCC` already ensures the alert type is enabled, we should be good to remove this.
```suggestion
```